Sources/Clickstream/EventScheduler/Domain/Protocols/FileStorable.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FileStorable.swift
+//  Clickstream
+//
+//  Created by Anirudh Vyas on 19/03/21.
+//  Copyright © 2021 Gojek. All rights reserved.
+//
+
+import Foundation
+
+protocol FileStorable {
```

Comment:
I guess this file is not needed? Can you where this is being used? Because we don't use CodableCache anymore

Reply:
Yes I removed this file